common/defs/bitcoin/horizen.json

```diff
@@ -1,16 +1,16 @@
 {
-  "coin_name": "Zencash",
+  "coin_name": "Horizen",
   "coin_shortcut": "ZEN",
-  "coin_label": "Zencash",
-  "website": "https://zencash.com",
+  "coin_label": "Horizen",
+  "website": "https://www.horizen.global",
   "github": "https://github.com/ZencashOfficial/zen",
-  "maintainer": "Zencash <support@zencash.com>",
+  "maintainer": "Power_VANO <vano@horizen.global>",
   "curve_name": "secp256k1",
   "address_type": 8329,
   "address_type_p2sh": 8342,
   "maxfee_kb": 2000000,
   "minfee_kb": 1000,
-  "signed_message_header": "Zencash Signed Message:\n",
+  "signed_message_header": "Horizen Signed Message:\n",
```

Comment:
Show me the exact place in the source code where this is defined. All I can find is this: https://github.com/ZencashOfficial/zen/blob/4ba3514e1ecf3c13c4ffe2f902aeb3e006f01b54/src/main.cpp#L109

Reply:
Ping

Reply:
After internal consultation seems "Zcash" is the correct one.

We suggest using the support@horizen.global email though so that alerts do not get missed in case I or someone else leaves the team. Is it ok for you?

Reply:
Let's keep the way it is. If you will leave the team, you'll send a PR.

Reply:
Ok.